### ISSUE TITLE
Import Suite: Blank window after adding all test cases in suite

### ISSUE THREAD

**Issue by clefley** [SeleniumHQ/selenium-ide#486]
## To Reproduce

1. Open Firefox
2. Click Selenium IDE toolbar icon
3. Click Open an existing project
4. Navigate to test suite repository folder
5. Display all files (*.*)
6. Select html file for suite
7. Click Open
8. Receive Import Suite dialog box
"Import suite
In order to fully import your legacy Selenium IDE suite, you need to individually import the following tests"
9. Either drag-and-drop or browse-navigate-select-open the requested test files.
10. Receive blank window with title bar "moz-extention://cc1bb99b-8e10-4c24-85b7-da3a5545ee6 - Mozilla Firefox".

## Expected behavior

Expected Selenium IDE to display the test suite with included test cases the Selenium IDE project window, ready to be executed. 

## Project file reproducing this issue (highly encouraged)

Attached Suite1.html and Case1.html

## Environment

OS: Windows 10
Selenium IDE Version:  3.4.4
Selenium SIDE Runner Version:  N/A
Please note that selenium-side-runner supports Node.js 8 - 10
-->
Node version: N/A
Only required if using selenium-side-runner
-->
Browser: Firefox
Browser Version: 62.0.3 32-bit

**Comment by clefley** [comment]
I opened this bug. When I attempt to use the Import Suite function I am prompted to import the individual tests but after adding them nothing processes. Only a blank screen with title bar "moz-extention://cc1bb99b-8e10-4c24-85b7-da3a5545ee6 - Mozilla Firefox" appears. I attached a Test Suite and Case but don't see any indication that they were attached. I am also unable to edit this bug report so I am adding this information as a comment.

**Comment by capoerafreak** [comment]
please check if the base URL is empty in your test scripts.
this is a known bug ...

**Comment by 98gmarquee** [comment]
@clefley You have to save it as a zip, then I think you can attach it by dragging/dropping it as you make a new comment.

**Comment by clefley** [comment]
Thanks, here you go.

**Comment by clefley** [comment]
Doesn't look like it's accepting zip files either.

**Comment by clefley** [comment]
Even trying to open the simple test case below (with no suite) I get this error.

     "Error migrating project
      n.html.head.link is undefined"

<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en" lang="en">
<head profile="http://selenium-ide.openqa.org/profiles/test-case">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
<title>Case1</title>
</head>
<body>
<table cellpadding="1" cellspacing="1" border="1">
<thead>
<tr><td rowspan="1" colspan="3">Case1</td></tr>
</thead><tbody>

<tr>
	<td>open</td>
	<td>https://www.google.com</td>
	<td></td>
</tr>
</tbody></table>
</body>
</html>

**Comment by clefley** [comment]
Let me try commenting it to see if the html displays


<tr>
	<td>open</td>
	<td>https://www.google.com</td>
	<td></td>
</tr>
</tbody></table>
</body>
</html>
-->

**Comment by clefley** [comment]
Guess not. I will need an address to send this to.

**Comment by Yrds** [comment]
Please edit your posts rather making double posts.

Could you Attach your .html again, please? I think it was not correctly uploaded.

**Comment by clefley** [comment]
I would have edited if I could but that wasn't working either. I was IE so I switched to Chrome and now I am able to edit and attach files. [Test Suite.zip](https://github.com/SeleniumHQ/selenium-ide/files/2599982/Test.Suite.zip)

**Comment by capoerafreak** [comment]
Please check the base URL at the html header if the base URL is empty this happens ...

**Comment by clefley** [comment]
The base URL is not defined in the html and there is no opportunity to enter a URL in the Se IDE because it closes before I can successfully open a suite and/or case.

**Comment by capoerafreak** [comment]
Maybee just open the html with an editor and enter a base URL ...

**Comment by clefley** [comment]
CONFIDENTIAL & RESTRICTED

What do I need to do to take advantage of this fix?

**Comment by tourdedave** [comment]
It will be available as part of the next release (date TBD). Or, you can build and install the IDE locally to get the fix now. Details on how to do that are in the readme: https://github.com/SeleniumHQ/selenium-ide/blob/master/README.md#prerequisites

--
Cheers,
Dave H

**Comment by tourdedave** [comment]
Yes, peru requires a POSIX machine. You could use a virtual machine to do the build, or manually do what peru does (e.g., copy files from one place to another). The `peru.yml` file in the project root specifies where things come from and where they go.

There are some breaking changes in master for the `select window` command that are not backwards compatible. If you're able to get a build working, you can use it to migrate your scripts and then switch to the current public version (3.4.4) to work with them. But if your scripts interact with multiple windows, then you'll need to wait until the next public release.

**Comment by tourdedave** [comment]
@clefley I discussed this with my team and we decided to do a point release to get this fix out there. Version 3.4.5 is available through Mozilla add-ons now.
